index.html

```diff
@@ -353,6 +353,35 @@ <h2>
               </tr>
             </tbody>
           </table>
+          <div class="note">
+            <p>Below are few non-normative recommendations when capturing display for 
+              applications that support full screen rendering.
+            </p>
+            <p>
+              For applications that support full screen rendering, like powerpoint for example, 
+              the decision of the display surface that gets captured while in full screen mode is 
+              specific to the user agent implementation. For consistency reasons, 
+              an user agent must ensure that the display surface captured in full screen 
+              mode be unambigously mapped to the application/window where the action was triggered 
+              in the first place. 
```

Comment:
The point here is for it to be clear that in a multi-window application the user agent is reasonably certain that the full screen switch was triggered by the window that was being captured.  You can point out here that this might be done by ensuring that this transition is only made when the captured window has focus before the transition.